src/lib/db.js

```diff
@@ -188,7 +188,7 @@ export class Database {
     });
   }
 
-  async connectActivityToCreator(url, creator) {
+  async connectLogToCreator(url, creator) {
```

Comment:
Sorry here was the function that should be renamed (see other comment).

Should also be rewritten as to not depend on `this.logActivity`:

```
let activity = new Activity(url);  // Will handle canocalization
activity.creator = a.url;
return await activity.save();
```

Reply:
Should I do that in this PR?

Reply:
@Powersource Yes.